bciers/apps/registration1/app/(onboarding)/home/page.tsx

```diff
@@ -66,174 +60,195 @@ export default function Page() {
   // Using breakpoints for responsive design, grid items will display in two columns on laptop & desktop and a single column on mobile & tablet,
   // Using the order prop for stacking order, grid item 1 and 2 will be reverse order on mobile & tablet,
   return (
-    <Grid
-      container
-      spacing={2}
-      sx={{
-        marginTop: "24px",
-        marginBottom: "48px",
-      }}
-    >
+    <>
+      <Note variant="important">
+        <div className="mt-4">
+          <b>Service Updates in Progress</b>
+        </div>
+        <div>
+          BCIERS is currently undergoing scheduled updates until mid-March.
+          During this time, user log-in will be temporarily unavailable. If you
+          require urgent support, please reach out to{" "}
+          <Link href={"mailto:GHGRegulator@gov.bc.ca"}>
```

Comment:
nit: We have hardcoded `mailto:GHGRegulator@gov.bc.ca` in a few places, but perhaps we could use `TILE_REPORT_PROBLEM` instead. However, this enum is found under the e2e/libs directory, which isn't particularly related.

Reply:
We have an urls file in libs, I'll pop it in there